src/hooks/useHandleSelectorComponent.tsx

```diff
@@ -15,12 +15,12 @@ export const useHandleSelectorComponent = ({ item, type }: { item: unknown; type
   }
 
   useEffect(() => {
-    if (type === SelectorType.STATUS_SELECTOR) {
+    if (type === SelectorType.STATUS_SELECTOR && item) {
```

Comment:
why add this check for `item`? can you add code comment here.

Reply:
`item` can be null and we don't want this block to run if it is null. That's why we are making a check here.